R/cut.R

```diff
@@ -204,12 +198,6 @@ cut_var <- function(var, breaks, include_outside_range) {
 # the levels when bake.recipe itself is called. Moreover,
 # it is cleaner to show it in this way.
 adjust_levels_min_max <- function(x) {
-  if (!is.factor(x)) {
-    cli::cli_abort(
-      "{.arg x} must be a factor, not {.obj_type_friendly {x}}.",
-      .internal = TRUE
-    )
-  }
```

Comment:
didn't feel like this was needed, since `adjust_levels_min_max()` is used on the output of `cut()` which should always be a factor.

I checked, and if `cut()` were to produce a non-factor, then other errors go off